dev-docs/documentation/contribute_docs.md

```diff
@@ -61,6 +61,18 @@ This will build *and serve* the documentation at http://127.0.0.1:8000 -- this
 gives you a live preview of any changes you make (and save) to the
 documentation without needing to continually rebuild.
 
+## Doc testing
+
+Before running the `make` command, make sure to download the python venv module by running the following command:
+```sh
+apt install python3-venv
+```
+When working with a `docs` branch and making updates, you can run the documentation tests using the Makefile. 
+The `make` command should list all targets to test. You can run individual targets using the command:
```

Comment:
It definitely *should* do that. Does it? (I don't usually run the tests, so not sure)

Reply:
Yes, when the `make` command is run, it lists all targets that can be run individually.

Reply:
Nice - in that case, could we tweak this to say the command "lists all targets" rather than "should list all targets"? I think it's a bit clearer then what the expected behaviour is